### DISCUSSION TITLE
Einstein Autocomplete for LWC Config File

### DISCUSSION THREAD

**Discussion by robertStrunk** [forcedotcom/Einstein-GPT-for-Developers#73] [General]
**Is your feature request related to a problem? Please describe.**
Nope, just a productivity boost

**Describe the solution you'd like**
Autocomplete for LWC XML config files. For example, after creating a new component I could navigate to the config file 
and type something like "record page" under the isExposed tag and it could buildout the targets > target necessary for record pages. 

**Describe alternatives you've considered**
- Looking up the documentation myself
- OR
- Creating a VSCODE snippet
- OR
- Using an external app like Lightning Studio to create LWCs
- OR
- Career swapping. I think I could disrupt the landscaping industry in my area
- OR
- Watching Cat videos all day on youtube until someone figures out I'm not being productive

**Additional context**
Single male almost 40 years old. I've drank a TON of Redbulls today. I like short walks on the beach. In a great mood. It's Friday and I just deployed a huge ADO ticket so I had a moment to come up for air. While up for air I decided to ask you guys for this feature. ⚡😎

**Comment by gbockus-sf** [comment]
@robertStrunk 🥇 on a well written feature request.  I'm going to covert to a discussion as that's where we're tracking enhancements.
